recipes/madminer/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "madminer" %}
+{% set version = "0.9.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/madminer-{{ version }}.tar.gz
+  sha256: 17eedd09da79db371530336904f248e8cde00032a53e50bbe36cbbb92f1c0939
+
+build:
+  # * PyTorch does not support Windows
+  # * uproot3-methods does not support Python 3.12+
+  # * Non noarch packages should have python requirement without any version constraints
+  # so use or statements in skip
+  skip: true  # [win or py<38 or py>=312]
```

Comment:
Can't use `noarch` as there are no distributions of `pytorch` for Windows, which means that can't place version restrictions on `python` in the requirements, so need to use the `skip` to restrict Python to 3.8+ but not allow Python 3.12.

Reply:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Ah okay I'll revert this and then tag you when I have this ready. :+1: